Makefile

```diff
@@ -207,3 +207,49 @@ check-torch-gpu: ## Check whether PyTorch works on GPU using VM with Custom Cont
 
 check-pipeline: ## Check whether the Beam pipeline can run on GPU using VM with Custom Container and DirectRunner
 	@./scripts/check-pipeline.sh
+
+create-flex-template: ## Create a Flex Template file using a Flex Template custom container
+	gcloud dataflow flex-template build $(TEMPLATE_FILE_GCS_PATH) \
+	--image $(CUSTOM_CONTAINER_IMAGE) \
+	--metadata-file ./flex/metadata.json \
+	--sdk-language "PYTHON" \
+	--staging-location $(STAGING_LOCATION) \
+	--temp-location $(TEMP_LOCATION) \
+	--project $(PROJECT_ID) \
+	--worker-region $(REGION) \
+	--worker-machine-type $(MACHINE_TYPE)
+
+run-df-gpu-flex: ## Run a Dataflow job using the Flex Template
+	$(eval JOB_NAME := beam-ml-starter-gpu-flex-$(shell date +%s)-$(shell echo $$$$))
+ifeq ($(MODEL_ENV), "TORCH")
+	gcloud dataflow flex-template run $(JOB_NAME) \
+	--template-file-gcs-location $(TEMPLATE_FILE_GCS_PATH) \
+	--project $(PROJECT_ID) \
+	--region $(REGION) \
+	--worker-machine-type $(MACHINE_TYPE) \
+	--additional-experiments disable_worker_container_image_prepull \
+	--parameters number_of_worker_harness_threads=1 \
+	--parameters sdk_location=container \
```

Comment:
`	--parameters sdk_location=container  `

has no effect on new SDKs.

Reply:
That's good. This was a really confusing to pass a sdk container and not getting used. Where did it change? Should we keep it here for now or remove?

Reply:
before beam was staging sdk code to staging locaiton; starting Beam 2.50.0 we don't do that, and assume beam SDK will be installed in container (the same behavior as when --sdk_location=container was specified for older SDKs).

Reply:
we should keep it now. Not all people use 2.50.0.